src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -326,4 +326,8 @@
   <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
     <Reference Include="System.Threading" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
Is this the correct condition, or should I add more? See the top of this same file, there are many windows condition combinations.